ammo-be/src/validators/bulletValidator.ts

```diff
@@ -0,0 +1,46 @@
+import Joi from 'joi';
+import Sizeof from 'object-sizeof';
+import urlRegex from 'utils/regexs/regexs';
+
+const customBodyLengthValidation = (value: string): string => {
+    // Sizeof calculations of bytes based
+    const requestBodySize = Sizeof(value);
+
+    // Arbitrary value of 50ko
+    if (requestBodySize > 50000) {
+        throw new Error('your body is too heavy, max 50Ko');
```

Comment:
Methods has no knowledge of any request. I'd suggest being transparent : 
requestBodySize => size
newError('Size exceeds the maximum 50ko authorized')